crates/modules/logger/src/lib.rs

```diff
@@ -49,51 +64,68 @@ impl TryFrom<&ModuleConfig> for Config {
         Ok(Self {
             console: config.with_default("console", true)?,
             // file: config.required("file")?,
-            // syslog: config.required("syslog")?,
+            syslog: config.with_default("syslog", true)?,
         })
     }
 }
 
+#[derive(Debug)]
 struct Logger {
     console: bool,
+    syslog: Option<UnixDatagram>,
 }
 
 impl Logger {
-    fn from_config(rx_config: Config) -> Self {
-        let Config { console } = rx_config;
-        Self { console }
+    fn from_config(rx_config: Config) -> Result<Self, Self> {
+        let Config { console, syslog } = rx_config;
+
+        let opt_sock = syslog
+            .then(|| {
+                let sock = UnixDatagram::unbound().ok()?;
+                UNIX_SOCK_PATHS
+                    .iter()
+                    .find_map(|path| sock.connect(path).ok())
+                    .map(|_| sock)
+            })
+            .flatten();
```

Comment:
You should handle the case when `pulsar` process is directly connected to the systemd journal to avoid a double log. In this case I think you can skip the manual connection, but you have to test it, I am not completely sure.

You can take a look [here](https://docs.rs/systemd-journal-logger/latest/systemd_journal_logger/fn.connected_to_journal.html)

Reply:
![Screenshot from 2023-10-12 13-45-33](https://github.com/Exein-io/pulsar/assets/86269947/1fd27422-eb72-4d85-ad31-d4b467c599ed)

Logs with pulsard ran as systemd service. Trying to log with color but displayed as raw text. There are no logs without attempted color interleaved, so I think it's working to prevent double logging.